wheatley/bot.py

```diff
@@ -137,7 +140,23 @@ def _on_row_gen_change(self, row_gen_json: JSON) -> None:
 
     def _on_size_change(self) -> None:
         self._check_number_of_bells()
+        self._opening_rounds = generate_starting_row(
+            self.number_of_bells, self.row_generator.custom_start_row)
         self._rounds = rounds(self.number_of_bells)
+        self._check_starting_row()
+
+    def _check_starting_row(self) -> bool:
+        if (self.row_generator.custom_start_row is not None and
+                len(self.row_generator.custom_start_row) < self.number_of_bells):
+            self.logger.info("The starting row contains fewer bells than the tower"
+                             + f"({self._tower.number_of_bells}) "
+                             + "Wheatly will add the extra bells to the end of the change")
```

Comment:
```suggestion
            self.logger.info(f"The starting row '{self.row_generator.custom_start_row}' "
                             + f"contains fewer bells than the tower ({self._tower.number_of_bells}). "
                             + "Wheatley will add the extra bells to the end of the change.")
```

Reply:
In _check_number_of_bells line 171 should check if there's a custom row

```py
if self._tower.number_of_bells > self.row_generator.stage + 1 and self.row_generator.custom_start_row is None:
```
Cover bells are added to `self._opening_rounds` by construction, with a message logged there when `custom_start_row` is set.

Currently the line 171 check incorrectly logs `-m "Plain Bob Doubles" --start-row 13572468`
> INFO:BOT:Current tower has more bells (8) than expected (6). Wheatley will add extra cover bells.

Reply:
Good catch